sdk/keyvault/azure-security-keyvault-secrets/inc/azure/keyvault/secrets/keyvault_secret_properties.hpp

```diff
@@ -21,6 +27,11 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Secrets {
    */
   struct SecretProperties final
   {
+    friend struct Azure::Security::KeyVault::Secrets::_detail::SecretSerializer;
+    friend struct Azure::Security::KeyVault::Secrets::_detail::
+        SecretPropertiesPagedResultSerializer;
+    friend struct Azure::Security::KeyVault::Secrets::_detail::DeletedSecretPagedResultSerializer;
+    friend struct Azure::Security::KeyVault::Secrets::_detail::SecretPropertiesSerializer;
```

Comment:
We can leave it as public.  It's not the SDK client responsibility to tell the customer what values can be set or not.  
The SDK client would just send the request to Azure and let the service decide what to do.

Reply:
makes sense